source/HailPanel.cpp

```diff
@@ -112,6 +117,8 @@ HailPanel::HailPanel(PlayerInfo &player, const shared_ptr<Ship> &ship, function<
 				helpOffer += "give you some fuel?";
 			else if(canRepair)
 				helpOffer += "patch you up?";
+			else if(canGiveEnergy)
+				helpOffer += "recharge you?";
```

Comment:
Noting that the help offer string doesn't account for all possible combinations of assistance to give. Also noting that I don't care.